bundle-size/github-utils.js

```diff
@@ -130,8 +130,8 @@ class GitHubUtils {
   /**
    * Get a list of all unique team members by team names.
    *
-   * @param {!Array<string>} teamNames names of full team slug names
-   *   ("org/team_slug") format to get members of.
+   * @param {!Array<string>} teamNames list of GitHub team (full) slugs,
```

Comment:
LGTM, would drop `(full)` since I don't think it adds anything esp with the examples. If you wanted to clarify it further, you could explicitly say org-namespaced slugs, but I think it's overkill.

Reply:
Dropped it.